doc/indexing.rst

```diff
@@ -478,6 +478,49 @@ Both ``reindex_like`` and ``align`` work interchangeably between
     # this is a no-op, because there are no shared dimension names
     ds.reindex_like(other)
 
+.. _multi-index handling:
+
+Multi-index handling
```

Comment:
These docs are great, but I wouldn't call them "indexing methods" exactly. Maybe move this section to [Reshaping and reorganizing data](http://xarray.pydata.org/en/stable/reshaping.html)?